### ISSUE TITLE
Support Ticket

### ISSUE THREAD

**Issue by viviancan** [Iridescent-CM/technovation-app#3892]

**Comment by shaunxp20** [comment]
> 3/28/23 Request around updating the thunkable url validation. See [thread here](https://iridescentlearning.slack.com/archives/C06CHBQ4E/p1680035315089369)

For reference, for a brief moment we had this enabled:
https://github.com/Iridescent-CM/technovation-app/issues?q=projectPage

And the convo around (reverting it):
https://iridescentlearning.slack.com/archives/C06CHBQ4E/p1618697310003200
https://iridescentlearning.slack.com/archives/C06CHBQ4E/p161869766900540

**Comment by viviancan** [comment]
3/28/23
Investigate location issues for completed submissions from Argentina. [Thread here ](https://iridescentlearning.slack.com/archives/C06CHBQ4E/p1680025574614199)

I am checking this out (VC)

**Comment by viviancan** [comment]
Update on the support issue -

TLDR of the issue: In the admin datagrid tables (teams and submissions) Argentina is displayed as the country for teams from `San Nicolás, Bío Bío, Chile`

After verifying in the DB, team location data appears to be correct. The team and submissions datagrids in the admin portal use `FriendlyCountry.new(team).country_name` to display the country name. So while the location data is correct in the DB something is happening with the friendly country method that is returning incorrect country data for certain teams in `San Nicolás, Bío Bío, Chile`. 

There is just 1 team (team id 30693) that shows the correct country (Chile) in the admin portal. I manually updated this team's location to `San Nicolas, Bío Bío, Chile` <-- without the accent on the `a` in `San Nicolas`. Moving forward the remaining teams could be manually updated through the admin portal or a note can be made for whoever is downloading this data. Reiterating that the data is correct in the DB, just the displayed country name in the data grids in the admin portal are incorrect for those teams. A list of those teams can [be found here](https://docs.google.com/spreadsheets/d/1vFOfSR7S1-FLB6xrf2JoquVUk-ISxANhRmShlynqMM0/edit?usp=sharing)

**Comment by viviancan** [comment]
test

**Comment by viviancan** [comment]
4/5/23

Teams not displaying for specific ChA even though the students are on teams. [See full context here](https://iridescentlearning.slack.com/archives/C06CHBQ4E/p1680709700859099)

**Comment by dboyer** [comment]
I met with Program Team to talk about the Thunkable link and we can keep it as /projects/ for this year. They will update the FAQ and Zendesk to respond to any questions. We may consider supporting /projectPage next year but no changes are needed to the Thunkable link for this year.

**Comment by dboyer** [comment]
@viviancan Tagging you since you've looked into a few of the last geocoding issues. We have a ChA who is reporting issues seeing participants in Madrid. On the participants page, the user is associated with the state listed as "Community of Madrid" but that is not a search option under state. I can find the user if I search by their name but if I add a state or city location filter, they are not returned as a search result.

![Screen Shot 2023-04-06 at 1.22.33 PM.png](https://images.zenhubusercontent.com/5ca619b412380971eb9c9477/b76ae1ec-660c-43ca-bd16-3fabef81d119)

I think this may be related to the name of the state changing in the geocoder. The state for newer registrants is listed as "Madrid, Comunidad de." I'm wondering if the participants from several years ago have a state listed for their location that is no longer searchable in the geocoder.
 ![Screen Shot 2023-04-06 at 1.26.04 PM.png](https://images.zenhubusercontent.com/5ca619b412380971eb9c9477/303f29f7-be44-4f5c-ac80-6580962744ae)

**Comment by viviancan** [comment]
@dboyer I will investigate the madrid issue ^^

**Comment by viviancan** [comment]
Opening a new ticket for the madrid issue #3910

**Comment by viviancan** [comment]
Reviewing submission data related to #3936 (icon not colored in for mentors even though submission has been submitted)

**Comment by dboyer** [comment]
Sent @viviancan an email regarding a judge who appears fully onboarded on her profile but is listed as "must complete onboarding" in the admin page for an RPE.

**Comment by dboyer** [comment]
Two submission issues:

1. A team submitted a project while one participant did not have a parental consent form completed. I'm wondering if that team had a pending invite out and the girl only accepted after the project was already submitted.

2. A team's submission is marked as submitted but is only 67% complete.

**Comment by viviancan** [comment]
Taking a look at the submission issues @dboyer @shaun-technovation ^^

**Comment by viviancan** [comment]
Following up on this issue

> Sent @viviancan an email regarding a judge who appears fully onboarded on her profile but is listed as "must complete onboarding" in the admin page for an RPE.

This issue is resolved. I believe the judge changed their email but did not confirm their new email address from the automated email that was sent to their account.. Sine their email had not been confirmed, they were marked false for the onboarded field in the DB, even though they had completed the onboarding steps (waiver & training). I wonder if we should add something to the admin debugging portal and the judge portal that tells them to please confirm their email. The field in the DB is `email_confirmed_at`

**Comment by viviancan** [comment]
> **A team submitted a project while one participant did not have a parental consent form completed. I'm wondering if that team had a pending invite out and the girl only accepted after the project was already submitted.**

Following up on this one it appears that student A submitted their completed project a `2023-04-19 00:54:39.548885` and then the invite for student B was updated at `2023-04-19 01:02:02.214879`. It looks like Student A submitted the project while the invite was still pending and then Student B accepted the invite.

After replicating the scenario locally it appears that Studnet B can accept the invite through the invite link from the email. If student B logs into their student portal there is not a way to accept the invite since they are still onboarding (missing parental consent).

------------------

> **A team's submission is marked as submitted but is only 67% complete.**

We think that the team may have submitted the project and then later deleted something from the submission that did not trigger the project to unsubmit. I was not able to recreate locally, but this season there were only 2 submissions that did not have 100% complete and were submitted. They have both been unsubmitted.

**Comment by viviancan** [comment]
Moving this to ready to deploy so we can have a fresh support ticket for the new iteration!
